### ISSUE TITLE
export - wrong column name

### ISSUE THREAD

**Issue by axzx** [sonata-project/SonataAdminBundle#4131]
#### Sonata packages

```bash
$ composer show sonata-project/*
sonata-project/admin-bundle              3.7.0 The missing Symfony Admin Generator
sonata-project/block-bundle              3.1.1 Symfony SonataBlockBundle
sonata-project/cache                     1.0.7 Cache library
sonata-project/core-bundle               3.1.1 Symfony SonataCoreBundle
sonata-project/doctrine-orm-admin-bundle 3.0.5 Symfony Sonata / Integrate Doctrine ORM into the SonataAdminBundle
sonata-project/exporter                  1.7.0 Lightweight Exporter library
```

#### Symfony packages

```bash
$ composer show symfony/*
symfony/monolog-bundle     2.11.1  Symfony MonologBundle
symfony/phpunit-bridge     v3.1.4  Symfony PHPUnit Bridge
symfony/polyfill-apcu      v1.2.0  Symfony polyfill backporting apcu_* functions to lower PHP versions
symfony/polyfill-intl-icu  v1.2.0  Symfony polyfill for intl's ICU-related data and classes
symfony/polyfill-mbstring  v1.2.0  Symfony polyfill for the Mbstring extension
symfony/polyfill-php56     v1.2.0  Symfony polyfill backporting some PHP 5.6+ features to lower PHP versions
symfony/polyfill-php70     v1.2.0  Symfony polyfill backporting some PHP 7.0+ features to lower PHP versions
symfony/polyfill-util      v1.2.0  Symfony utilities for portability of PHP codes
symfony/security-acl       v3.0.0  Symfony Security Component - ACL (Access Control List)
symfony/swiftmailer-bundle v2.3.11 Symfony SwiftmailerBundle
symfony/symfony            v3.1.4  The Symfony PHP framework
```

#### PHP version

```bash
$ php -v
PHP 7.0.8-0ubuntu0.16.04.2 (cli) ( NTS )
Copyright (c) 1997-2016 The PHP Group
Zend Engine v3.0.0, Copyright (c) 1998-2016 Zend Technologies
    with Xdebug v2.4.0, Copyright (c) 2002-2016, by Derick Rethans
    with Zend OPcache v7.0.8-0ubuntu0.16.04.2, Copyright (c) 1999-2016, by Zend Technologies
```

## Subject
In export should be taken the value of the label.

## Steps to reproduce
```php
protected function configureListFields(ListMapper $listMapper)
{
    $listMapper->add('activeCode', null, array(
        'label' => 'Code',
    ));
}
```

## Expected results
In export csv file, column name should be "Code" (translated of course).

## Actual results
In export csv file, column name is "activeCode".

**Comment by greg0ire** [comment]
Did you have a look at #3856 ?

**Comment by axzx** [comment]
@greg0ire 
yes. could you explain me?

**Comment by greg0ire** [comment]
>  yes. could you explain me?

I don't know what you want explained, but I would like to know if you have that particular commit. Also, I advise you to debug this piece of code, and to query the symfony profiler with the url of the export action : you should see if there are missing translations (maybe you're not using the right catalog, etc…)

**Comment by axzx** [comment]
I checked it.
It looks like it was not considered an option `'label' => 'Code'`.
Is created label `Active Code` instead of `Code`.

Maybe @core23 knows what is wrong?

**Comment by greg0ire** [comment]
> Maybe @core23 knows what is wrong?

Probably, but I think he might be away for a while…

> It looks like it was not considered an option 'label' => 'Code'.
Is created label Active Code instead of Code.

I don't understand your sentence, sorry, can you rephrase / elaborate ?

**Comment by axzx** [comment]
I thought it will be used an option `label` from `ListMapper`.
Probably I miscalculated;)

**Comment by greg0ire** [comment]
Oh ok I get it… well this is clearly not how it works. The documentation in the linked PR seems pretty clear to me, but if you feel it lacks some sentences, please make a documentation PR.
